src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataExtractor.java

```diff
@@ -28,7 +29,27 @@ private ScannedDocumentsExtractor() {
 
     public static List<ScannedDocument> getScannedDocuments(Envelope envelope) {
         List<Document> documents = envelope.documents;
-        return documents.stream().map(ScannedDocumentsExtractor::mapDocument).collect(toList());
+        return documents.stream().map(CaseDataExtractor::mapDocument).collect(toList());
+    }
+
+    @SuppressWarnings("unchecked")
+    public static Map<String, String> getOcrData(CaseDetails caseDetails) {
+        List<Map<String, Object>> ccdOcrData =
+            (List<Map<String, Object>>) caseDetails.getData().get("scanOCRData");
+
+        if (ccdOcrData != null) {
+            return ccdOcrData
+                .stream()
+                .map(ccdCollectionElement -> ((Map<String, String>) ccdCollectionElement.get("value")))
+                .collect(
+                    toMap(
+                        map -> map.get("key"),
+                        map -> map.get("value")
+                    )
+                );
+        } else {
+            return null;
```

Comment:
perhaps an empty map would be safer?

Reply:
Given we're only verifying data from CCD (not sending to them), can be done.